dfx.json

```diff
@@ -59,7 +61,8 @@
 			"build": "scripts/build.cycles_depositor.sh",
 			"init_arg_file": "out/cycles_depositor.args.did",
 			"wasm": "out/cycles_depositor.wasm",
```

Comment:
Don't the wasm files need to `.gz` extension?

Reply:
Good question.  I don't know.  It works now but why, when the downloaded file is `.wasm.gz`? `dfx` magically doing the right thing?  Let me investigate.

Reply:
Thanks! I wouldn't spend too much time investigating, there are other priorities... It must be dfx doing some magic.